cmd/root.go

```diff
@@ -62,6 +62,7 @@ would otherwise be a time consuming manual task. It also wraps docker compose
 commands to manage the lifecycle of stacks.
 
 To get started run: ff init
+Optional: Set FIREFLY_HOME env variable for Firefly stack configuration path.
```

Comment:
```suggestion
Optional: Set FIREFLY_HOME env variable for FireFly stack configuration path.
```

For branding consistency, we capitalize both F's in places that are "user facing".